src/Paket/Program.fs

```diff
@@ -49,6 +49,9 @@ type CLIArguments =
     | [<AltCommandLine("-f")>] Force
     | Hard
     | [<CustomCommandLine("nuget")>] Nuget of string
+//    | [<CustomCommandLine("gist")>] Gist of string * string
+//    | [<CustomCommandLine("github")>] Github of string * string
+    | [<CustomCommandLine("url")>] Url of string * string
     | [<CustomCommandLine("version")>] Version of string
     | [<CustomCommandLine("add-credentials")>] AddCredentials of string
```

Comment:
| [<CustomCommandLine("gist")>] MultiGist of string * string
    | [<CustomCommandLine("gist")>] SingleGist of string
Does not work either. `paket add gist http://..` works, but if I provide a file name, it throws saying that the parameter is not recognized. The parser seems to work with SingleGist all the time. Don't know that else to try. @forki could you look into this when you have a moment?